homeassistant/components/zha/core/channels/security.py

```diff
@@ -343,8 +343,9 @@ def cluster_command(self, tsn, command_id, args):
         """Handle commands received to this cluster."""
         if command_id == 0:
             state = args[0] & 3
-            self.async_send_signal(
-                f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", 2, "zone_status", state
+            # update attribute cache (sends signal)
+            self.cluster.update_attribute(
+                IasZone.attributes_by_name["zone_status"].id, state
```

Comment:
I don't think the "cooked" value should go in the cache. The value reported by the device should.

Reply:
We probably improperly do this in a few spots unfortunately

Reply:
Removed all "bit-stripping" from the `IASZoneChannel`. It's only done in the `IASZone` binary sensor now (`parse()` method).

Like before the changes from this PR, this will likely still call `async_write_ha_state()` for the binary sensor even if one of the relevant bits didn't change.
As this was already the case before (it doesn't really matter that much), if that needs to be cleaned up, I'd try to do that in a separate PR. And with that, maybe clean up the `attribute_updated` methods in all different channels and move the check for specific attributes up to the entity level.